### DISCUSSION TITLE
Custom.scss file showing error

### DISCUSSION THREAD

**Discussion by kavigithub** [twbs/bootstrap#39042] [Q&A]
Hello,

I am trying to add custom.scss file for my project . But i got this error. Can someone please look into it. I am not able to figure out where where i am doing wrong. Please check with the attachment.

<img width="1397" alt="image" src="https://github.com/twbs/bootstrap/assets/16161263/1a61c478-b401-4b82-aad6-74eeb5661711">
<img width="767" alt="image" src="https://github.com/twbs/bootstrap/assets/16161263/0cac7ed8-a407-466e-8b36-8eee1d62781a">
<img width="686" alt="image" src="https://github.com/twbs/bootstrap/assets/16161263/1f43607d-6a5b-4b7e-ab59-5e0705454147">

**Comment by JamesVanWaza** [comment]
@kavigithub  try 
so in `style.scss` add `@import "custom"`;

<img width="604" alt="Screenshot 2023-08-13 at 8 43 17 PM" src="https://github.com/twbs/bootstrap/assets/4000076/ec59248d-facf-46d0-8f80-6d3963ba70d4">

<img width="344" alt="Screenshot 2023-08-13 at 8 43 29 PM" src="https://github.com/twbs/bootstrap/assets/4000076/80d2ffc2-463e-4fba-bb47-269710f69b22">


Try with the screnshots and let me know

**Reply by kavigithub** [reply]
hey @JamesVanWaza...this is my react.js project
<img width="1217" alt="image" src="https://github.com/twbs/bootstrap/assets/16161263/c73cc9a9-b83b-4b6c-8436-17a768a2fba6">

i added the functions and variables scss in my index.scss but still i got that error. But still i got the same error. You can check SS.

**Reply by patkon** [reply]
You don't need to import the bootstrap functions if you import all of bootstrap afterwards anyway.

so this ...
```
// variable overrides
@import "../node_modules/bootstrap/scss/bootstrap";
// youre othe scss/css stuff
```
... is enough.

**Reply by kavigithub** [reply]
I did that ...i mean i removed it...but still got the same error...

**Reply by kavigithub** [reply]
<img width="1155" alt="image" src="https://github.com/twbs/bootstrap/assets/16161263/c7e7e1ea-c647-492d-a790-035edc22091a">
<img width="1179" alt="image" src="https://github.com/twbs/bootstrap/assets/16161263/0799b529-24c9-4fdc-8717-51560a9d597e">
